extensions/tokenizers/rust/src/compute_cap.rs

```diff
@@ -0,0 +1,96 @@
+use candle::cuda_backend::cudarc::driver::sys::CUdevice_attribute::{
+    CU_DEVICE_ATTRIBUTE_COMPUTE_CAPABILITY_MAJOR, CU_DEVICE_ATTRIBUTE_COMPUTE_CAPABILITY_MINOR,
+};
+use candle::cuda_backend::cudarc::driver::CudaDevice;
+use std::sync::Once;
+
+static INIT: Once = Once::new();
+static mut RUNTIME_COMPUTE_CAP: usize = 0;
+static mut COMPILE_COMPUTE_CAP: usize = 0;
+
+fn init_compute_caps() {
```

Comment:
Need implement RsEngine.hasCapability() using this function.

Reply:
Added